timeseries-aggregator/src/main/resources/config/base.conf

```diff
@@ -21,3 +21,5 @@ kafka {
   }
 }
 health.status.path = "./isHealthy"
+
+haystack.graphite.address = "haystack.local"
```

Comment:
should the default graphite address be the same as the one we use for jmxtrans-agent ? 
"monitoring-influxdb-graphite.kube-system.svc"